services/src/main/java/org/keycloak/truststore/FileTruststoreProviderFactory.java

```diff
@@ -77,6 +77,10 @@ public void init(Config.Scope config) {
         String policy = config.get(HOSTNAME_VERIFICATION_POLICY);
         String configuredType = config.get("type");
 
+        if (storepath != null || pass != null || configuredType != null) {
+            log.warn("Using deprecated truststore-spi options. Consider using truststore-paths option.");
```

Comment:
I'd be more explicit in this case as we already know the specific provider and format of these options. WDYT?
```suggestion
            log.warn("Using deprecated 'spi-truststore-file-*' options. Consider using 'truststore-paths' option.");
```

I'm ok with not introducing additional complexities here.